### DISCUSSION TITLE
Change how to access web gui from dns fqdn to ip?

### DISCUSSION THREAD

**Discussion by jonvassmer** [Security-Onion-Solutions/securityonion#5105] [Unsupported Versions] [answered]
I setup security onion to use DNS hostname, but now I want to change access to IP. Is there a way to do that?
thanks in advanced. I am on version 2.3.61
Jon

**Comment by weslambert** [comment, marked as answer]
You should be able to do this with the instructions here: https://docs.securityonion.net/en/latest/url-base.html
